lib/main_common.pm

```diff
@@ -45,6 +45,10 @@ our @EXPORT = qw(
   load_extra_tests
   load_rollback_tests
   load_filesystem_tests
+  load_fips_tests_core
+  load_fips_tests_web
+  load_fips_tests_misc
+  load_fips_tests_crypt
```

Comment:
the file `lib/main_common.pm` is meant for test calls common for SLE+openSUSE. It would be great to also have FIPS-tests for openSUSE if that is meaningful but I don't think this was your current intention so better move these helper functions to products/sle/main.pm